pages/guides/developing/viewer-engagement.en-US.mdx

```diff
@@ -8,23 +8,61 @@ import { Tab, Tabs } from 'nextra-theme-docs';
 
 # Explore Viewer Engagement
 
+Livepeer offers detailed information on viewer behavior and playback quality on your platform. The API includes engagement metrics such as view counts and watch time, as well as performance metrics such as error rate, time to first frame, rebuffer ratio, and exit-before-starts across a variety of dimensions.
+
+### Performance Metrics
+
+This API offers several key performance metrics to help you understand your platforms performance:
+
+- Error Rate: Percentage of views that encountered an error.
+- Time to First Frame (TTFF): Measures the time in milliseconds it takes between the player requesting video and the first frame being ready to play.
+- Rebuffer Ratio: Percentage of time a viewer spends experiencing rebuffering during playback.
+- Exit Before Starts: Percentage of views that are abandoned before the video begins to play, potentially indicating issues with playback.
+
+### Usage Metrics
+
+This API also includes usage metrics to help you understand how viewers are engaging with your platform:
+
+- Number of Views: This metric gives you a comprehensive view of the total number of views across your platform. We colloquially define a view as “play intent”, which means the video either played, stalled, or encountered an error.
+- Hours of Watch Time: Measure the total amount of time viewers spend on your platform, so you can track engagement and identify areas for improvement.
+
+### Dimensions
+
+To provide a comprehensive view of viewer behavior, our data product includes several dimensions to help you segment your data and identify patterns:
+
+- Video: View metrics for individual videos, queryable by playbackId or by dStorage Id’s such as CID.
```

Comment:
nit: `Id's` -> `identifiers` or `IDs`?

Reply:
isnt ID short for identifier? Happy to change it to whichever one people would expect to see here.

Reply:
yeah I'm just nitpicking on capitalization of the "d" in ID